audformat/core/utils.py

```diff
@@ -936,9 +940,26 @@ def join_schemes(
           labels: [a, b]
 
     """
-    labels = join_labels([db.schemes[scheme_id].labels for db in dbs])
-    for db in dbs:
-        db.schemes[scheme_id].replace_labels(labels)
+    is_misc_table = np.array(
+        [isinstance(db.schemes[scheme_id].labels, str) for db in dbs],
+    )
+    if is_misc_table.any():
+        if not is_misc_table.all():
```

Comment:
Maybe it makes more sense we check for this case in `join_labels()` and raise the error there?

Reply:
`join_labels()` doe snot support misc tables yet. Would you add support for it there as well?
Then you don't need to update `join_schemes()` at all maybe.

Reply:
We cannot support it there because we need access to the database to get the misc table. But we can at least check if a user tries to combine a list of labels with labels from a misc table.

Reply:
OK, but then it will not help here as `join_labels()` should not be called when joining schemes with misc tables.

Reply:
> But we can at least check if a user tries to combine a list of labels with labels from a misc table.

We also state 
```python
labels: typing.Sequence[typing.Union[typing.List, typing.Dict]],
```
in `join_labels()` so a user is not supposed to provide a string (and hence a misc table).
So far we have not raised errors for cases that do not fit the typing.
But maybe it would be helpful to remind the user in this case.

Reply:
But I think I get it now, you will use the first `if`-statement only if all tables are mics tables and otherwise handover to `join_labels()` which should then raise an error if any misc table is left, correct?

Reply:
Yes, I updated it accordingly.

Reply:
> in join_labels() so a user is not supposed to provide a string (and hence a misc table).
So far we have not raised errors for cases that do not fit the typing.
But maybe it would be helpful to remind the user in this case.

Good point, in principle we do not have to consider this case. But now we raise a specific error message if a misc table ID is passed.